BlueprintUI/Sources/Element/Element+Debug.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+
+extension Element {
+
+    @_spi(Debugging)
+    public func debugScope(_ scope: String) -> Element {
```

Comment:
We should probably come back to this at some point—I don't know if we gave it enough though (maybe it's fine though?).